--- code/syntax.php~	2009-01-28 15:50:10.000000000 +0200
+++ code/syntax.php	2009-01-28 15:51:08.389855693 +0200
@@ -10,7 +10,7 @@
 	// Include parent class:
 	require_once(DOKU_PLUGIN . 'syntax.php');
 	// We're dealing with "GeSHi" here, hence include it:
-	require_once(DOKU_INC . 'inc/geshi.php');
+	require_once '/usr/share/php/geshi.php';
 
 /**
  * <tt>syntax_plugin_code.php </tt>- A PHP4 class that implements the
@@ -864,7 +864,7 @@
 					$aRenderer->doc .= '</pre>';
 				} else {
 					$isSH = ('bash' == $aData[2]);
-					$geshi = new GeSHi($aData[1], $aData[2], GESHI_LANG_ROOT);
+					$geshi = new GeSHi($aData[1], $aData[2]);
 					if ($geshi->error()) {
 						// Language not supported by "GeSHi"
 						$geshi = NULL;	// release memory
@@ -998,7 +998,7 @@
 						$aData[3], $aData[0]);
 				} else {
 					$isSH = ('bash' == $aData[2]);
-					$geshi = new GeSHi($aData[1], $aData[2], GESHI_LANG_ROOT);
+					$geshi = new GeSHi($aData[1], $aData[2]);
 					if ($geshi->error()) {
 						// Language not supported by "GeSHi"
 						$geshi = NULL;	// release memory
